code/Card.tsx

```diff
@@ -121,20 +140,28 @@ export class Card extends React.Component<Props> {
                         width: `100%`,
                     }}
                     {...ComponentProps}
-                    cover={
-                        <img
-                            src={imgSrc}
-                        />
+                    actions={
```

Comment:
I was trying to get elements of the card component to show on the canvas after turning options on in the config but didn't have much luck yet